src/components/hmi_message_handler/test/CMakeLists.txt

```diff
@@ -28,35 +28,40 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-if(BUILD_TESTS)
+include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/platform.cmake)
+include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/sources.cmake)
 
 include_directories (
     ${GMOCK_INCLUDE_DIRECTORY}
     ${COMPONENTS_DIR}/hmi_message_handler/include
     ${COMPONENTS_DIR}/hmi_message_handler/test/include
 )
 
+if (HMIADAPTER STREQUAL "messagebroker")
+  set (BROKER_LIBRARIES
+    message_broker_client
+    message_broker_server
+  )
+endif()
+
+set(EXCLUDE_PATHS)
+
 set(LIBRARIES    
     gmock
     ApplicationManager
     HMIMessageHandler
     jsoncpp
-    MessageBrokerClient
-    MessageBrokerServer
+	${BROKER_LIBRARIES}
```

Comment:
@AByzhynar BROKER_LIBRARIES just remains blank, so it will not cause any change to the library list